src/lib/tagHelper.ts

```diff
@@ -0,0 +1,20 @@
+const hasEventBubbledThroughTag = ({
+  tagName,
+  target,
+  currentTarget,
```

Comment:
consider passing the event over target/currentTarget separately

Reply:
The reason why I send those as separate params is beacuse this is a recursive function and in the subsequent calls, for the target prop we send the target's parent.